Sources/Plot/API/HTMLComponents.swift

```diff
@@ -578,6 +578,62 @@ extension List: ComponentContainer where Items == ComponentGroup {
         self.init(content()) { $0 }
     }
 }
+// Component used to render a `<picture>` element for responsive image support.
+public struct Picture: Component {
+
+    /// Image to use as fallback.
+    public var image: Image
+    /// A set of `source` elments from which a user agent will chose based on its requirements.
+    public var sources: [Source]
+
+    /// Initialize a `Picture` component with multiple `source` elements.
+    internal init(image: Image, sources: [Picture.Source] = []) {
+        self.image = image
+        self.sources = sources
+    }
+
+    /// Initialize a `Picture` component with a single `source` element.
+    /// Image to use as fallback.
+    /// A set of `source` elments from which a user agent will chose based on its requirements.
+    public init(image: Image, source: Source) {
+        self.init(image: image, sources: [source])
+    }
+
+    public var body: Component {
+            Node.picture(
```

Comment:
Sill incorrect indentation here. Please format your code according to the rest of the library (4 spaces indentation per level) to keep things consistent.